### DISCUSSION TITLE
Change Monitoring code  to django commands

### DISCUSSION THREAD

**Discussion by petrunetworking** [eduardogsilva/routerfleet#33] [Ideas]
Change Monitoring code and status of router to django command using example python3 manage.py monitord or another name of command it will be much more convenient but also easier to start and stop the monitoring

**Comment by eduardogsilva** [comment]
Hello @petrunetworking thanks for the suggestion.

The monitoring feature is a standalone docker container that consumes the routerfleet monitoring api. 

Can you detail your use case please?

cheers!
